processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
I think this should be changed because a join query can involve multiple data sources of different intervals. The intervals for each data source now can be emitted by calling ```dataSources()```. Please see ```DataSourceWithSegmentSpec```.